### PR TITLE
[WIP #71] - Use build stages for docs deployment

### PR DESCRIPTION
**Pull request by mforets**

### CODE REVIEW

**Comment by schillic** [comment]
Yeah, it worked!

**Comment by mforets** [comment]
let me try again.. it failed because of a `ERROR: Documenter is not an installed package` in the in the new "deploy docs" stage.

**Comment by schillic** [comment]
Damn, now it worked?

**Comment by schillic** [comment]
@mforets: Since you asked back in the other thread: I think their use case was to fail the build when building the docs fails. This is different from the doc**tests**.

**Comment by mforets** [comment]
then i don't know what else to try  😢

**Comment by schillic** [comment]
My guess is that Documenter just prints warnings. Travis does not recognize them as an error.

Maybe this can help?
```
Documenter: skipping docs deployment.
You can set DOCUMENTER_DEBUG to "true" in Travis to see more information.
```

**Comment by schillic** [comment]
I just found [this](https://github.com/JuliaDocs/Documenter.jl/wiki/Documenter.jl-roadmap#use-travis-build-stages). Not sure if this is what you did...

**Comment by mforets** [comment]
i would suggest to close this in favor of #467

**Comment by schillic** [comment]
Yes. Do we want to use the idea with build stages there as well?

**Comment by mforets** [comment]
i never got the build stages to work as expected as evidenced in our comments above  :/

**Comment by schillic** [comment]
Okay, then I will close.
